README.md

```diff
@@ -110,7 +110,7 @@ EASYPOST_TEST_API_KEY=123... EASYPOST_PROD_API_KEY=123... mvn clean test -B jaco
 
 ### Testing
 
-The test suite in this project was specifically built to produce consistent results on every run, regardless of when they run or who is running them. <s>This project uses [EasyVCR]() to record and replay HTTP requests and responses via "cassettes". When the suite is run, the HTTP requests and responses for each test function will be saved to a cassette if they do not exist already and replayed from this saved file if they do, which saves the need to make live API calls on every test run.</s>
+The test suite in this project was specifically built to produce consistent results on every run, regardless of when they run or who is running them. <s>This project uses [EasyVCR](https://github.com/EasyPost/easyvcr-java) to record and replay HTTP requests and responses via "cassettes". When the suite is run, the HTTP requests and responses for each test function will be saved to a cassette if they do not exist already and replayed from this saved file if they do, which saves the need to make live API calls on every test run.</s>
```

Comment:
I mean, it doesn't use EasyVCR yet. How did this get here in the first place?

Reply:
We copied in the whole section when we started working on it months ago. It is stiked through though in the README itself. We'll be shortly adopting it